### PR TITLE
feat: allow Nullable targetElement

### PR DESCRIPTION
**Pull request by magic-akari**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [x] Other, please describe: Types

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following platforms:

- [ ] PC
- [ ] iOS
- [ ] Android

Providing a reproduction demo will be better(you can pick a platform below and **fork** the demo):

* jsbin: https://jsbin.com/cafiful/edit?output
* codepen: https://codepen.io/buptsteve/pen/PvNQjP
* jsfiddle: https://jsfiddle.net/buptsteve/6u8g3Lf5/
* codesandbox: https://codesandbox.io/s/o73z4jy5q9

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

Why Nullable: 

`document.querySelector()` or `document.getElementById()` return Nullable result.

Writing `lock(document.querySelector('.target'))` is more concise than writing `lock(document.querySelector('.target') || undefined)`

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
 
 type OverflowHiddenPcStyleType = 'overflow' | 'boxSizing' | 'paddingRight'
 type OverflowHiddenMobileStyleType = 'top' | 'width' | 'height' | 'overflow' | 'position'
+type Nullable<T> = T | null;
```
Comment (BuptStEve) [inline src/index.ts]: no semicolon

**Comment by BuptStEve** [comment]
LGTM

File: src/index.ts (hunk 1 above)
Comment (magic-akari) [reply src/index.ts]: > no semicolon

done

**Comment by BuptStEve** [comment]
@all-contributors add @magic-akari for bug

**Pull request merged**